### DISCUSSION TITLE
New tag: #astro

### DISCUSSION THREAD

**Discussion by ismaxs** [dailydotdev/daily#1336] [Feature Requests]
[https://astro.build/](https://astro.build/)

**Comment by idoshamun** [comment]
Hi @ismaxs,

here you go https://app.daily.dev/tags/astro

**Reply by ismaxs** [reply]
Thanks @idoshamun ,

So why can't I find it in the 'new feed' section?

**Reply by idoshamun** [reply]
I just added it now upon your request :)

**Reply by ismaxs** [reply]
Oh, I'm sorry, I didn't understand it in the first message. Thank you very much.

**Reply by idoshamun** [reply]
no worries i never mentioned that 😅
